cmd/root.go

```diff
@@ -102,9 +102,8 @@ func Execute(cmd *cobra.Command, args []string) {
 	// Clean up a previous container
 	podmanRemove(types.PivotName)
 
-	// `podman mount` wants a running container, so let's make a dummy one
-	cid := utils.RunGetOutln("podman", "run", "-d", "--net=none", "--name",
-		types.PivotName, "--entrypoint", "sleep", imgid, "infinity")
+	// `podman mount` wants a container, so let's make create a dummy one, but not run it
+	cid := utils.RunGetOutln("podman", "create", "--net=none", "--name", types.PivotName, imgid)
```

Comment:
Yeah, this is *way* better.

Reply:
I also want to note that new oscontainer build process here: https://github.com/coreos/coreos-assembler/pull/90#issuecomment-423297992

*requires* this change in pivot.

Reply:
Though actually to work it also needs to do `--entrypoint /enoent`, like:
https://github.com/coreos/coreos-assembler/pull/90/files#diff-d02610260c957c8f46e86e93faf93cfdR41

Since you have a dev/test environment set up for this, mind doing that?

Reply:
Wait, can you expand why it needs that? Is it because there will no longer be a default entrypoint embedded in the container and it's invalid to have neither a cmd nor entrypoint?

I.e. does it makes sense instead to do something like `buildah config --entrypoint /enoent` instead when building it? That reinforces the idea that this container is only ever meant to be `create`d, not `run`.